docs/source/en/api/pipelines/aura_flow.md

```diff
@@ -62,6 +62,34 @@ image = pipeline(prompt).images[0]
 image.save("auraflow.png")
 ```
 
+Loading [GGUF checkpoints](https://huggingface.co/city96/AuraFlow-v0.3-gguf) are also supported:
```

Comment:
I think linking to the more general GGUF guide is more useful than the checkpoint itself

```suggestion
Loading [GGUF checkpoints](https://huggingface.co/docs/diffusers/quantization/gguf) are also supported:
```